### PR TITLE
Jaeger deprecation

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Added warnings concerning jaeger deprecation
- Added links to the jaeger installation tutorial
- Edited images related to jaeger usage in the system

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/16390

### CODE REVIEW

File: docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Every runtime provides its own unique environment configuration which can be rea
 
 > **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 use OpenTelemetry protocol-compliant endpoint, as defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (a-thaler) [inline docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md:27]: That caution is not needed here. Serverless switched already to the generic approach, see caution above

File: docs/04-operation-guides/operations/02-install-kyma.md [hunk 2]
```diff
@@ -68,6 +68,9 @@ A profile is defined globally for the whole Kyma installation. It's not possible
 
 If you install Kyma on a remote cluster, you can use the out-of-the box `kyma.example.com` domain. All you need to do is get your load balancer IP address and add the following line to the `hosts` file:
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (chrkl) [inline docs/04-operation-guides/operations/02-install-kyma.md:71]: The warning feels a bit misplaced here since the rest of the document is not related to Jaeger. I think just removing it from the list of hostnames below should be fine. New users should not start using the deprecated Jaeger, but deploy their own one immediately.

File: docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 title: Telemetry - TracePipeline
 ---
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (chrkl) [inline docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md:5]: Same is before: This is not related to Jaeger. Just change the endpoint in the example below.

File: docs/05-technical-reference/svls-08-function-specification.md [hunk 4]
```diff
@@ -112,6 +112,8 @@ See the detailed descriptions of these fields:
 | **tracer** | Fully configured OpenTelemetry [tracer](https://opentelemetry.io/docs/reference/specification/trace/api/#tracer) object that allows you to communicate with the Jaeger service to share tracing data. For more information on how to use the tracer object see [Use the OpenTelemetry standard](../03-tutorials/00-serverless/svls-12-use-opentelemetry-client.md) |
 | **extensions** | JSON object that can contain event payload, a Function's incoming request, or an outgoing response |
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (chrkl) [inline docs/05-technical-reference/svls-08-function-specification.md:115]: I think the warning is also not required here. Maybe change the description of the `tracer` field in a way that an arbitrary OpenTelemetry endpoint can be used.

**Review by chrkl — CHANGES_REQUESTED** [review]

**Review by chrkl — APPROVED** [review]

**Review by grego952 — APPROVED** [review]
